charts/consul/templates/enterprise-license-job.yaml

```diff
@@ -1,5 +1,6 @@
 {{- if (or (and (ne (.Values.server.enabled | toString) "-") .Values.server.enabled) (and (eq (.Values.server.enabled | toString) "-") .Values.global.enabled)) }}
-{{- if (and .Values.server.enterpriseLicense.secretName .Values.server.enterpriseLicense.secretKey (not .Values.server.enterpriseLicense.enableLicenseAutoload)) }}
+{{- if (and .Values.global.enterpriseLicense.secretName .Values.global.enterpriseLicense.secretKey (not .Values.global.enterpriseLicense.enableLicenseAutoload)) }}
+{{- if .Values.server.enterpriseLicense }}{{ fail "server.enterpriseLicense has been moved to global.enterpriseLicense" }}{{ end -}}
```

Comment:
Could we put the fail at the top? And then maybe we only need one for the entire chart.

Reply:
AAH! ok that makes sense. It should simplify the PR.

Reply:
Should the bats test for this check also reside only for that file or does it make sense to test that change in all the files that utilize the license? 🤔 
I see the argument in doing both.

Reply:
I think the pattern we've followed is only for that file. Let me check.

Reply:
Yeah, I think that's the pattern.

Reply:
🙏 thanks!!